app/models/facility.rb

```diff
@@ -130,5 +135,15 @@ def diabetes_enabled?
     enable_diabetes_management.present?
   end
 
+  def teleconsultation_enabled?
+    enable_teleconsultation.present?
+  end
+
   CSV::Converters[:strip_whitespace] = ->(value) { value.strip rescue value }
+
+  private
+
+  def clear_isd_code
+    self.teleconsultation_isd_code = ""
```

Comment:
Any difference between setting to `""` and `nil`? How does the validation messaging behave when we try to save a facility with isd code but no number?

Reply:
Not a big difference, since `nil.present?` and `"".present?` are both false. Just sticking to the defaults, all the other text fields in our forms get saved with a `""` value when they're empty.

Reply:
![Screenshot 2020-05-11 at 3 28 04 PM](https://user-images.githubusercontent.com/48791115/81549279-197c9e00-939c-11ea-8b60-16658ec66f70.png)